### ISSUE TITLE
Detect more page patterns

### ISSUE THREAD

**Issue by mskelton** [playwright-community/eslint-plugin-playwright#131]
There are some more instances were we aren't detecting page usage.

```
foo.page()...
const page2 = ...
const otherPage = ...
```
